main.go

```diff
@@ -45,12 +47,13 @@ const depNixFormat = `  {
 func getModules() ([]*modEntry, error) {
 	var entries []*modEntry
 
-	commitShaRev := regexp.MustCompile(`^v\d+\.\d+\.\d+-(?:\d+\.)?[0-9]{14}-(.*?)$`)
+	commitShaRev := regexp.MustCompile(`^v\d+\.\d+\.\d+-(?:\d+\.)?[0-9]{14}-(.*?)(?:\+incompatible)?$`)
```

Comment:
In my case, I have this version string which don't match with this regex: `v0.34.0-rc3.0.20200907055413-3359e0bf2f84`.

Reply:
Which project has such version numbers?

Reply:
```suggestion
	commitShaRev := regexp.MustCompile(`^v\d+\.\d+\.\d+-(?:rc\d+\.)?(?:\d+\.)?[0-9]{14}-(.*?)(?:\+incompatible)?$`)
```

Reply:
> Which project has such version numbers?

https://github.com/cosmos/cosmos-sdk/blob/master/go.mod#L43

Reply:
the modified regex above works for me.